src/modals/confirmation-modal/operations-confirmation/operations-confirmation.tsx

```diff
@@ -48,6 +53,10 @@ export const OperationsConfirmation: FC<Props> = ({ sender, opParams, isLoading,
   } = useFeeForm(opParams, estimations.data);
 
   const handleSubmit = ({ gasFeeSum, storageLimitSum }: FeeFormInputValues) => {
+    if (opParams[0]?.kind === OpKind.DELEGATION && opParams[0]?.delegate === recommendedBakerAddress) {
+      trackEvent('EVERSTAKE_BAKER_DELEGATION', AnalyticsEventCategory.FormSubmit);
```

Comment:
naming mismatch between `recommendedBakerAddress` and `EVERSTAKE_BAKER_DELEGATION`